src/main/java/org/sagebionetworks/bridge/sdk/BridgeUserClient.java

```diff
@@ -121,13 +129,14 @@ public void changeSharingScope(SharingScope sharingScope) {
     }
 
     @Override
-    public void withdrawConsentToResearch(String reason) {
+    public void withdrawConsentToResearch(SubpopulationGuid subpopGuid, String reason) {
         session.checkSignedIn();
+        checkNotNull(subpopGuid, CANNOT_BE_NULL, "subpopGuid");
         
         Withdrawal withdrawal = new Withdrawal(reason);
-        post(config.getWithdrawConsentSignatureApi(), withdrawal);
-        session.setSharingScope(SharingScope.NO_SHARING);
-        session.setConsented(false);
+        post(config.getWithdrawConsentSignatureApi(subpopGuid), withdrawal);
+        session.setSharingScope(SharingScope.NO_SHARING); // is this still true?
```

Comment:
Nit: Could you please sign/date TODO questions? I know that there are only a couple of us, but it helps when/if others come onboard.